frontend/packages/client/src/helpers/theme/index.js

```diff
@@ -1,14 +1,44 @@
+//
+// ***********************************
+// Component style overrides for Chakra components
+import Input from './components/Input';
+import Link from './components/Link';
+import Text from './components/Text';
+// ***********************************
+//
+// ***********************************
+// Custom components style definitions
+import { CardStyle } from 'components/Card';
+// ***********************************
 import { extendTheme } from '@chakra-ui/react';
+//
+// ***********************************
+// Global definitions overrides
+import definitions from './definitions';
+// ***********************************
+//
+// Foundational style overrides
+import borders from './foundations/borders';
+// ***********************************
+//
+// ***********************************
+// Global style overrides
+import styles from './styles';
 
-const theme = extendTheme({
-  styles: {
-    global: (props) => ({
-      'html, body': {
-        bg: 'transparent',
-        lineHeight: 'normal',
-      },
-    }),
+// ***********************************
+
+const overrides = {
+  ...definitions,
+  styles,
+  borders,
+  // Other foundational style overrides go here
+  components: {
+    Input,
+    Card: CardStyle,
+    Link,
+    Text,
+    // Other components go here
   },
-});
+};
 
-export default theme;
+export default extendTheme(overrides);
```

Comment:
this file provides configuration to use a custom theme with chakra

`definitions.js` file overwrites default definitions by Chakra to match designs
`components` folder contains one file per component we are overwriting to match designs, by having one file per component it's easy to find and keeps things more isolated 
`foundations` folder contains definitions for borders, radius, typography (currently we are using bulma and it should be ported here)
`styles` file defines global styles for html elements
there's more to add but this is the basic structure